packages/vitest/src/node/logger.ts

```diff
@@ -212,4 +213,22 @@ export class Logger {
     }))
     this.log(c.red(divider()))
   }
+
+  async printAsyncLeaksWarning(hangingOps: HangingOps[]) {
+    if (hangingOps.length === 0)
+      return
+    const errorMessage = c.yellow(c.bold(
+      `\nVitest has detected the following ${hangingOps.length} hanging operation${hangingOps.length > 1 ? 's' : ''} potentially keeping Vitest from exiting: \n`,
```

Comment:
> <ins>potentially</ins> keeping Vitest from exiting

This is well phrased. 👍 

None of these patterns (timers, promises) should prevent Vitest from exiting as we don't do graceful exits - or have you so far noticed any hangs? But this will still be useful for finding leaks from tests and source code.

Reply:
Thanks for the feedback! Actually this phrase is ported from Jest and I thought it may can be used in Vitest as well. 

I haven't found explicit hangings in my daily use, but yeah I think people may benefit from this feature and am willing to be involved.